src/test/scala/linting/rule/LintConflictingModuleNamesSpec.scala

```diff
@@ -0,0 +1,189 @@
+// See LICENSE for license details.
+
+package freechips.rocketchip.linting.rule
+
+import firrtl._
+import firrtl.ir.Port
+import firrtl.annotations._
+import firrtl.testutils.{FirrtlMatchers, FirrtlPropSpec}
+
+case class StableNameAnnotation(target: IsModule) extends SingleTargetAnnotation[IsModule] {
+  def duplicate(newTarget: IsModule): StableNameAnnotation = this.copy(target = newTarget)
+}
+
+case class UnstableNameAnnotation(target: IsModule) extends SingleTargetAnnotation[IsModule] {
+  def duplicate(newTarget: IsModule): UnstableNameAnnotation = this.copy(target = newTarget)
+}
+
+class RenameDesiredNamesSpec extends FirrtlPropSpec with FirrtlMatchers {
+  val transform = new RenameDesiredNames
+
+  case class TestCase(
+    input: String,
+    annos: Seq[Annotation]
+  )
+
+  def stabilizeNames(testCase: TestCase): CircuitState = {
```

Comment:
should this just be called "renameDesiredNames"? I don't really know what "stabilizeNames" means